Review user guide (#979)
ansible/receptor: docs/source/roadmap/receptor_roadmap_index.rst

Receptor Roadmaps
=================

> The ``Receptor`` team develops a roadmap for each major  ``Receptor`` release.

Reviewer: ```suggestion
The ``Receptor`` team develops a roadmap for each major ``Receptor`` release.
```

Reviewer: Any reason Receptor is wrapped in backticks here? Seems inconsistent with elsewhere in the doc.

Author: No, it's removed

The latest roadmap shows current work; older roadmaps provide a history of the
project.  We don't publish roadmaps for minor or subminor versions. So 2.0 and
3.0 have roadmaps, but 2.1.0 and 2.10.1 do not.

We incorporate team and community feedback in each roadmap, and aim for further
transparency and better inclusion of both community desires and submissions.

Each roadmap offers a *best guess*, based on the ``Receptor`` team's experience
and on requests and feedback from the community, of what will be included in a
given release.  However, some items on the roadmap may be dropped due to time
constraints, lack of community maintainers, and so on.

Each roadmap is published both as an idea of what is upcoming in ``Receptor``,
and as a medium for seeking further feedback from the community.

> Feedback can be submitted on the current roadmap by:

Reviewer 2: ```suggestion
You can submit feedback on the current roadmap by:
```

> - Creating issue on github in ansible/receptor

Reviewer: ```suggestion
- Creating issue on GitHub in the ``ansible/receptor`` repository.
```

Reviewer: Perhaps add a link to the repository here.

Author: done, found some other places and added there also
